Terminal.Gui/Application/Application.Driver.cs

```diff
@@ -26,7 +26,7 @@ public static partial class Application // Driver abstractions
     ///     Note, <see cref="Application.Init(IConsoleDriver, string)"/> will override this configuration setting if called
     ///     with either `driver` or `driverName` specified.
     /// </remarks>
-    [SerializableConfigurationProperty (Scope = typeof (SettingsScope))]
+    [ConfigurationProperty (Scope = typeof (SettingsScope))]
```

Comment:
Is the reason we use typeof instead of enum so API users can add their own scopes? I guess string with `nameof` would also work... just interested not really something i think needs changed.